src/mono/mono/metadata/marshal.c

```diff
@@ -3688,13 +3688,19 @@ mono_marshal_get_native_func_wrapper_indirect (MonoClass *caller_class, MonoMeth
 	g_assert (!sig->hasthis && ! sig->explicit_this);
 	g_assert (!sig->is_inflated && !sig->has_type_parameters);
 
+#if 0
+	/*
```

Comment:
How can non-blittable types be passed to unmanaged code, how are they marshalled ?

Reply:
(I'm going by what I see in https://github.com/dotnet/runtime/issues/46402#issuecomment-756486124)

It works the same way as a normal pinvoke: the wrapper does marshalling based on the managed signature.  we don't know the exact callee, but we know the types.